### ISSUE TITLE
FEATURE - Large Widgets, View Maintenance Events

### ISSUE THREAD

**Issue by mikaelacaron** [mikaelacaron/Basic-Car-Maintenance#66]
## Describe the feature you want the app to have that it currently doesn't
* Create a large widget that allows you to view the total number of maintenance events for a specific vehicle
* Be able to configure which vehicle you want to view for this widget

## Describe alternatives you've considered
N/A
## Additional context
* Only make the large widget, there's different issues for different widget sizes

**Comment by Dishant10** [comment]
I would like to work on this @mikaelacaron. Please assign this to me. 
Thank you!

**Comment by ayushs-2k4** [comment]
I would like to work on this feature @mikaelacaron.

**Comment by mikaelacaron** [comment]
@Dishant10  you commented first, it's yours!
@ayushs-2k4 there's no need to comment on issues that someone else has already asked to take, and it hasn't been assigned yet. Please look for another issue, thank you!

**Comment by mikaelacaron** [comment]
Sorry! This task isn't ready to be worked on right now, because I haven't connected the models together, feel free to choose another issue and I'll leave you as an assignee and let you know when you can come back to work on this issue

**Comment by Dishant10** [comment]
Got it! Will wait for new issues as well, because all the open issues have been assigned.

**Comment by mikaelacaron** [comment]
> Got it! Will wait for new issues as well, because all the open issues have been assigned.

Thank you for waiting! I'll be adding new issues today / tomorrow. I'll post in the [Discussions](https://github.com/mikaelacaron/Basic-Car-Maintenance/discussions) tab in announcements when I've added new ones

**Comment by mikaelacaron** [comment]
Please wait on #141 and once you see that's merged, you can begin this issue, thanks for your patience!

**Comment by mikaelacaron** [comment]
This can be worked on now! Thanks so much for the patience

**Comment by mikaelacaron** [comment]
Actually sorry, hold off again, this isn't saving correctly

**Comment by mikaelacaron** [comment]
Thanks for your patience! I'm going to hold off on all the widget tasks for right now because I'm going to change how the vehicles and maintenance events are connected. But because it's so close to the end of Hacktoberfest I don't want to break everything for everyone, so I'll open this issue again after Hacktoberfest and you're welcome to contribute to it for fun if you'd like

**Comment by mikaelacaron** [comment]
Opening this back up for Hacktoberfest 2024!

**Comment by hanhqvu** [comment]
@mikaelacaron 
I know that this is labeled `good first issue` but could I take on this😀

**Comment by mikaelacaron** [comment]
yeah you got it! `good first issue` is to indicate skill level, that this is an overall easier task to get started with!

`first timers only` is really the only one that I really watch for, so that it's only people who have never contributed to the repo yet!

**Comment by hanhqvu** [comment]
@mikaelacaron 
sorry for the lack of update, I was a bit busy with work
I saw that the person working on the medium widget already worked on integrating data fetching so I will focus on the UI
after their medium widget PR is merged, I will integrate later
